src/Data/Connection/AbstractConnectionResolver.php

```diff
@@ -589,7 +588,7 @@ public function get_ids() {
 	 * @return int|mixed
 	 */
 	public function get_offset() {
-			_deprecated_function( __FUNCTION__, '1.9.0', get_class( $this ) . '::get_offset_for_cursor()' );
+		_deprecated_function( __FUNCTION__, '1.9.0', get_class( $this ) . '::get_offset_for_cursor()' ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
Should this be escaped instead of ignored?

Reply:
Nvm, it seems fine since `get_class( $this )` isnt subject to abuse.

https://github.com/WordPress/WordPress-Coding-Standards/blob/develop/WordPress/Tests/Security/EscapeOutputUnitTest.inc#L131